### PR TITLE
Hosting `credentialSchema`

### PR DESCRIPTION
**Pull request by EbonyLouis**
To help us get [this PR](https://github.com/TBD54566975/developer.tbd.website/pull/1453) out I hosted our Known Customer Credential Schema on our website. 

## Issue
Closing out [this](https://github.com/TBD54566975/known-customer-credential/issues/37) issue

## Direct Link To Preview

https://deploy-preview-1456--tbd-website-developer.netlify.app/schemas/kccSchema.json

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207707995611628

### CODE REVIEW

File: site/static/schemas/kccCredentialSchema.json
Comment (angiejones) [inline site/static/schemas/kccCredentialSchema.json:1]: nit: the last `c` in kcc stands for credential so kccCredential is repetitive. suggest: `kccSchema.json`

**Review by angiejones — APPROVED** [review]

File: site/static/schemas/kccCredentialSchema.json
Comment (EbonyLouis) [reply site/static/schemas/kccCredentialSchema.json:1]: LOL true - I need a nap

**Comment by nitro-neal** [comment]
oh this is huge, currently I have one hosted on a random website.

As soon as this goes in I'll update mine to point to tbds and make sure it is working correctly
https://github.com/TBD54566975/web5-js/pull/737

**Comment by nitro-neal** [comment]
and by the way, 

for us to use this link programmatically, there needs to be a url that resolves to raw json. like this:
https://purple-charming-snail-690.mypinata.cloud/ipfs/QmZbPpfPXsp4bFQvQvWRaexCu9Vxmj6qkwWerVfPwY9kQS

image

and cant have header and stuff:

image

File: site/static/schemas/kccSchema.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          "id": {
+            "type": "string"
+          },
+          "country_of_residence": {
+            "type": "string",
+            "pattern": "^[A-Z]{2}$"
+          },
+          "tier": {
+            "type": "string",
+            "optional": true
```
Comment (decentralgabe) [inline site/static/schemas/kccSchema.json:17]:
```suggestion
            "optional": true
```
```suggestion
            "type": "string"
```

**Comment by EbonyLouis** [comment]
@nitro-neal I renamed the file so the link needed to be updated it. Check this one out theres no heading: https://deploy-preview-1456--tbd-website-developer.netlify.app/schemas/kccSchema.json

File: site/static/schemas/kccSchema.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+          }
+        },
+        "required": [
+          "id",
+          "country_of_residence"
```
Comment (decentralgabe) [inline site/static/schemas/kccSchema.json:22]:
```suggestion
          "countryOfResidence"
```

File: site/static/schemas/kccSchema.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+      "credentialSubject": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string"
+          },
+          "country_of_residence": {
```
Comment (decentralgabe) [inline site/static/schemas/kccSchema.json:11]:
```suggestion
          "countryOfResidence": {
```

File: site/static/schemas/kccSchema.json [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+      },
+      "credentialSchema": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "format": "uri"
+          },
+          "type": {
+            "type": "string"
```
Comment (decentralgabe) [inline site/static/schemas/kccSchema.json:44]:
```suggestion
            "type": "string",
            "const": "JsonSchema"
```

File: site/static/schemas/kccSchema.json [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+            "checks": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          },
+          "optional": true
+        },
+        "optional": true
```
Comment (decentralgabe) [inline site/static/schemas/kccSchema.json:69]:
```suggestion
          }
        },
```

**Review by decentralgabe — COMMENTED** [review]
Left some comments.

This looks good as a starting point, though I imagine we will want to update this schema to include other properties such as the credential id, issuer, issued at time, etc. in the future.

**Comment by EbonyLouis** [comment]
Thanks! @decentralgabe

**Pull request merged**